back-end/db/friendsTriviaQuestions.ts

```diff
@@ -0,0 +1,146 @@
+export default [
```

Comment:
trivia about the TV show Friends (and one about Seinfeld) - this way, no player is the subject of the final question, which could cause issues